packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -200,19 +159,22 @@ export default function NavigableToolbar( {
 	focusOnMount,
 	focusEditorOnEscape = false,
 	shouldUseKeyboardFocusShortcut = true,
-	__experimentalInitialIndex: initialIndex,
-	__experimentalOnIndexChange: onIndexChange,
 	...props
 } ) {
 	const toolbarRef = useRef();
 	const isAccessibleToolbar = useIsAccessibleToolbar( toolbarRef );
 
+	if ( focusOnMount ) {
```

Comment:
It is undocumented, but, yes, it is available as [a prop on the `<NavigableToolbar />`](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/navigable-toolbar/index.js#L200).

Reply:
I see it documented in the [README.md](https://github.com/WordPress/gutenberg/blob/66e087114f3452f9ad310c9f31a6325b6faef700/packages/block-editor/src/components/navigable-toolbar/README.md). Not sure if it should be entirely removed from the docs there or marked as deprecated.

Reply:
Seeing as it was documented, we should probably add back the code to cause the focus in case someone was using that. It can be removed after the deprecation period.

Reply:
> Talking about [this focus](https://github.com/WordPress/gutenberg/pull/56335/files?w=1#diff-597e80ea2dbb7e8c5056536c126e0b0d54c4dcc79c6bd6ba1aaad35e1539c56cL147-L152). Is it still needed?

That code was related to placing on the passed `__experimentalInitialIndex` to focus a specific button on mount. The [`focusOnMount` code is still there](https://github.com/WordPress/gutenberg/pull/56335/files#diff-597e80ea2dbb7e8c5056536c126e0b0d54c4dcc79c6bd6ba1aaad35e1539c56cR120-R124).

Reply:
> I see it documented in the [README.md](https://github.com/WordPress/gutenberg/blob/66e087114f3452f9ad310c9f31a6325b6faef700/packages/block-editor/src/components/navigable-toolbar/README.md). Not sure if it should be entirely removed from the docs there or marked as deprecated.

On second thought, the deprecation notice in the docs is definitely the way to go. I like the format from https://github.com/WordPress/gutenberg/blob/c516601bad06a5b8dbe78f84148afb94471ddd4b/packages/block-editor/src/components/url-popover/README.md#position.